settings.d/dynflow.yml.example

```diff
@@ -1,3 +1,5 @@
 ---
 :enabled: true
 :database: /var/lib/foreman-proxy/dynflow/dynflow.sqlite
+:dynflow_console_authorize_with_ssl_client: false
```

Comment:
Also, since it's a dynflow plugin, we can assume, that the console is dynflow as well, so we can shorten the option to `console_authorize_with_ssl_client`